Lumos_prepopulated_tables.sql

```diff
@@ -0,0 +1,49 @@
+-- create stements for all 3 tables in the case users, resources, and images.
+CREATE TABLE users (
+	id serial primary key,
+	name varchar(150) NOT NULL,
+	email varchar(120) NOT NULL,
+	phone_number varchar(18),
+	password_hash varchar(72) NOT NULL
+);
+
+CREATE TABLE resources (
+	id serial primary key,
+	location geometry NOT NULL,
+	type varchar(60) NOT NULL,
+	name varchar(120),
+	quantity int NOT NULL CHECK (quantity > 0),
+	owned_by int,
+	reserved_by int,
+	foreign key (owned_by) references users(id),
+	foreign key (reserved_by) references users(id)
+);
+
+CREATE TABLE images (
+	id serial primary key,
+	content varchar(300) NOT NULL,
+	resource_id int,
+	foreign key (resource_id) references resources(id)
+);
+
+create index resources_gix on resources using GIST (location);
+
```

Comment:
```suggestion
```
We don't need these create table statements since we do that in the first migration

Reply:
sounds good I was just making sure too include them but I will remove them.